apps/core/src/dashboards/dashboards.service.ts

```diff
@@ -29,4 +29,9 @@ export class DashboardsService {
   public async delete(id: Dashboard['id']) {
     return this.repository.delete(id);
   }
+
+  public async bulkDelete(ids: Dashboard['id'][]) {
+    // TODO: use BatchWriteItem instead of Promise.all
```

Comment:
Is there anything preventing us from using BatchWriteItem now instead of later?

Reply:
Not really. It's more work. I'm pro-iterative development. We can do this change in a follow up.

Reply:
If we are doing batch, we will want to make sure each dashboard's summary and definition are deleted together. `TransactWriteItems` would probably be the safer API to use so the deletion is atomic. However, it has a limit of 100 items so we would want to keep that in mind ([doc](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_TransactWriteItems.html#DDB-TransactWriteItems-request-TransactItems))

Reply:
If we are calling the batch API, we need to keep in mind of the atomicity. We want to make sure each dashboard's summary and definition are delete together. [TransactWriteItems](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_TransactWriteItems.html) might be the better API; however, keep in mind of its [limit of 100 items](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_TransactWriteItems.html#DDB-TransactWriteItems-request-TransactItems) and constraint of no partial update/deletion.